R/plotting.R

```diff
@@ -1087,3 +1082,41 @@ CloneConnGraph <- function(object, reduction, group.by = NULL, groups.highlight
 
   plot + geom_node_point()
 }
+
+#' Use Seurat's DEG Table to plot a volcano plot
+#'
+#' @param data Table with DEG data from Seurat::Findmarkers()
+#' @param c.scheme colorscheme to use
+#'
+#' @export
+
+VolcanoPlotDEG <- function(data, c.scheme = c("coolwarm", "viridis")) {
+    # For coloring: adding a colomn to indicate diff expression
+    data$expression <- "zero"
+    data$expression[data$avg_log2FC > 0.6 & data$p_val_adj < 0.05] <- "up"
+    data$expression[data$avg_log2FC < -0.6 & data$p_val_adj < 0.05] <- "down"
```

Comment:
Maybe add these values as parameters?

The p-values in single-cell data don't actually have a real meaning. Most of the p-values will be very low, but this often doesn't mean they are significant. P-values should only be used to rank your genes relative to each other. 
Doing things like this can therefor be 'dangerous' since they can indicate a lot of things as 'upregulated' even though this is not really the case.